diff --git a/packages/web-runtime/src/extensionPoints.ts b/packages/web-runtime/src/extensionPoints.ts
index cc9990a2bb..fe4198ac81 100644
--- a/packages/web-runtime/src/extensionPoints.ts
+++ b/packages/web-runtime/src/extensionPoints.ts
@@ -31,6 +31,12 @@ export const progressBarExtensionPoint: ExtensionPoint<CustomComponentExtension>
   }
 }
 
+export const snackbarExtensionPoint: ExtensionPoint<CustomComponentExtension> = {
+  id: 'app.runtime.snackbars',
+  extensionType: 'customComponent',
+  multiple: true
+}
+
 export const topBarCenterExtensionPoint: ExtensionPoint<CustomComponentExtension> = {
   id: 'app.runtime.header.center',
   extensionType: 'customComponent',
@@ -55,6 +61,7 @@ export const extensionPoints = () => {
       appMenuExtensionPoint,
       preferencesPanelExtensionPoint,
       progressBarExtensionPoint,
+      snackbarExtensionPoint,
       topBarCenterExtensionPoint,
       topBarLeftExtensionPoint,
       topBarRightExtensionPoint
diff --git a/packages/web-runtime/src/layouts/Application.vue b/packages/web-runtime/src/layouts/Application.vue
index 4776034b60..70665f9aba 100644
--- a/packages/web-runtime/src/layouts/Application.vue
+++ b/packages/web-runtime/src/layouts/Application.vue
@@ -38,6 +38,7 @@
     >
       <message-bar />
       <upload-info />
+      <custom-component-target :extension-point="snackbarExtensionPoint" />
     </div>
   </div>
 </template>
@@ -59,7 +60,7 @@ import UploadInfo from '../components/UploadInfo.vue'
 import { useRouteMeta, useSpacesLoading, useNavItems } from '@opencloud-eu/web-pkg'
 import { computed, nextTick, onMounted, unref } from 'vue'
 import { useGettext } from 'vue3-gettext'
-import { progressBarExtensionPoint } from '../extensionPoints'
+import { progressBarExtensionPoint, snackbarExtensionPoint } from '../extensionPoints'
 import AppFloatingActionButton from '../components/AppFloatingActionButton.vue'
 
 const { $gettext } = useGettext()
